uwsgi/admin.wsgi

```diff
@@ -22,15 +22,15 @@ DOMAIN_WHITELIST = {
     "clients2.googleusercontent.com": ("Widevine",),
     "redirector.gvt1.com": ("Widevine",),
     "ftp.mozilla.org": ("SystemAddons",),
-    "fpn.firefox.com": ("Guardian",),
+    "fpn.firefox.com": ("FirefoxVPN", "Guardian"),
 }
 if STAGING or LOCALDEV:
     SYSTEM_ACCOUNTS.extend(["balrog-stage-ffxbld", "balrog-stage-tbirdbld"])
     DOMAIN_WHITELIST.update(
         {
             "ftp.stage.mozaws.net": ("Firefox", "Fennec", "Devedition", "SeaMonkey", "Thunderbird"),
             "bouncer-bouncer-releng.stage.mozaws.net": ("Firefox", "Fennec", "Devedition", "SeaMonkey", "Thunderbird"),
-            "stage.guardian.nonprod.cloudops.mozgcp.net": ("Guardian",),
+            "stage.guardian.nonprod.cloudops.mozgcp.net": ("FirefoxVPN", "Guardian"),
```

Comment:
I wonder if the URLs will change as well at some point to encompass the new name

Reply:
Yeah, that's a good question. I think that's @jbuck's department. Doesn't matter to me though, especially for stage.